### DISCUSSION TITLE
Implementation of the Orange-Boosted-Bootstrap timePicker

### DISCUSSION THREAD

**Discussion by laulilaura** [Orange-OpenSource/Orange-Boosted-Bootstrap#2481] [Q&A] [answered]
Hello,
I'm looking to implement an Orange-Boosted-Bootstrap standard time picker, but system.design.orange is out of order on this point and I can't find anything in this linked git about time-pickers or date-pickers. Is this component still available and if so, how can I implement it?
Thanks a lot!

**Comment by julien-deramond** [comment, marked as answer]
Hello @laulilaura 
There's no implementation of time/date pickers in Boosted apart from basic styles of `<input type="datetime-local">`,  `<input type="time">` and `<input type="date">`.
In order to implement what's described in [ODS > Web > Forms > Design](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459), you would have to find an external date/time library compatible with the stack of your project.
For instance, with Angular, you could use [ng-bootstrap Datepicker](https://ng-bootstrap.github.io/#/components/datepicker/overview) + some CSS style.
What's your stack, frameworks, etc.?

**Reply by laulilaura** [reply]
OK, thanks, I'll manage, that answers my question!
